example/geocodes/geocodes.dart

```diff
@@ -254,15 +245,12 @@ class _GeocodesApp extends react.Component {
 
     // Prepare the URL
     addressQuery = Uri.encodeQueryComponent(addressQuery);
-    var path =
-        'https://maps.googleapis.com/maps/api/geocode/json?address=$addressQuery';
+    var path = 'https://maps.googleapis.com/maps/api/geocode/json?address=$addressQuery';
 
     // Send the request
-    HttpRequest.getString(path)
-        .then((String value) =>
-            // Delay the answer 2 more seconds, for test purposes
-            new Future.delayed(new Duration(seconds: 2), () => value))
-        .then((String raw) {
+    HttpRequest.getString(path).then((String value) =>
+        // Delay the answer 2 more seconds, for test purposes
+        new Future.delayed(new Duration(seconds: 2), () => value)).then((String raw) {
```

Comment:
I know you can't do anything about this, but i hate that it does the same methods on the same line.... grosss. would using a cascade operator change it? no right?

Reply:
Cascade would change it, but I'll just update it to use async/await